docs/writing-analysis-jobs.md

```diff
@@ -0,0 +1,126 @@
+# How to extend Cartography with Analysis Jobs
+
+## Intro
+In a nutshell, Analysis Jobs let you add your own customizations to Cartography by writing Neo4j queries.  This helps you add powerful enhancements to your data without the need to write Python code.
+
+### The stages
+There are 3 stages to a cartography sync.  First we create database indexes, next we ingest AWS assets, and finally we can run Analysis Jobs on the database (see [cartography.sync.build\_default\_sync()](https://github.com/lyft/cartography/blob/master/cartography/sync.py)).  This tutorial focuses on Analysis Jobs.
+
+### How to run
+Each Analysis Job is a JSON file with a list of Neo4j statements which get run in order.  To run Analysis Jobs, in your call to `cartography`, set the `--analysis-job-directory` parameter to the folder path of your jobs.  Although the order of statements within a single job is preserved, different jobs will run in a different order each time.
```

Comment:
"different jobs will run in a different order each time" it's more accurate to say we don't guarantee the order in which jobs are executed. They probably will run in the same order each time on most platforms.